classes/class-handler.php

```diff
@@ -263,6 +263,8 @@ public function get_kses_for_allowed_blocks( array $tags ) {
 		$tags['mark'] = [ 'class' => true ];
 		$tags['sub'] = [];
 		$tags['sup'] = [];
+		$tags['span'] = [ 'style' => true ];
```

Comment:
Note: I'm fairly sure there's been some kses changes around having `style` validation, but I'm not sure if this is 100% safe without validating that functionality does actually exist.